connectors/sources/gcs.py

```diff
@@ -17,7 +17,15 @@
 
 from connectors.logger import logger
 from connectors.source import BaseDataSource
-from connectors.utils import TIKA_SUPPORTED_FILETYPES, get_base64_value
+from connectors.utils import (
+    DEFAULT_MAX_CONCURRENCY,
+    DEFAULT_QUEUE_MEM_SIZE,
```

Comment:
do we really want to reuse the bulk queue default params? I think we want a specific set of default values for `gcs` here